modules/cluster-logging-configuration-of-json-log-data-for-default-elasticsearch.adoc

```diff
@@ -79,32 +79,36 @@ Suppose that you use the following snippet in your `ClusterLogForwarder` CR YAML
 
 [source,yaml]
 ----
```

Comment:
outputs:
  - name: default
    elasticsearch:
      structuredTypeKey: openshift.labels.test
      structuredTypeName: nologformat
pipelines:
  - name: application-logs
    inputRefs:
    - application
    - audit
    outputRefs:
    - elasticsearch-secure
    - default
    parse: json
    labels:
      test: test

Reply:
openshift.labels.test is the label defined in pipeline.  not in the source code.

Reply:
Made updates as discussed in slack thread: https://redhat-internal.slack.com/archives/GGUR75P60/p1718034341293289